docs/pages/distribution/runtime-versions.mdx

```diff
@@ -46,10 +46,22 @@ For an iOS build, add an entry to the **Expo.plist** with the key `EXUpdatesRunt
 + <string>2.718</string>
 ```
 
-For an Android build, add a `<meta-data>` element to the **AndroidManifest.xml** whose `android:name` attribute is `expo.modules.updates.EXPO_RUNTIME_VERSION` and `android:value` attribute is a string that represents the desired runtime version.
+For an Android build, add an entry to **android/app/src/main/res/values/strings.xml** with the name `expo_runtime_version`. The value is a string that represents the runtime version.
+
+```diff android/app/src/main/res/values/strings.xml
+--- a/android/app/src/main/res/values/strings.xml
++++ b/android/app/src/main/res/values/strings.xml
+@@ -1,3 +1,4 @@
+ <resources>
+     <string name="app_name">MyApp</string>
++    <string name="expo_runtime_version">2.718</string>
+ </resources>
+```
```

Comment:
Change is not required, code blocks are fine, but we also have a `DiffBlock` component, which formats the snippets a bit differently, example:
* https://docs.expo.dev/bare/installing-expo-modules/#configuration-for-android